.github/workflows/release-dry-run.yml

```diff
@@ -5,19 +5,20 @@ on: push
 jobs:
   release-dry-run:
     runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.PAT }}
     steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-          token: ${{ secrets.PAT }}
-      - name: Install Knope
-        run: |
-          wget https://github.com/knope-dev/knope/releases/download/$KNOPE_VERSION/knope-x86_64-unknown-linux-musl-$KNOPE_VERSION.tgz
-          tar -xvf knope-x86_64-unknown-linux-musl-$KNOPE_VERSION.tgz
-          cp knope-x86_64-unknown-linux-musl-$KNOPE_VERSION/knope $HOME/.cargo/bin
-        env:
-          KNOPE_VERSION: v0.4.1
-      - name: Display Pending Release
-        run: knope release --dry-run
-        env:
-          GITHUB_TOKEN: ${{ secrets.PAT }}
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        token: ${{ secrets.PAT }}
+    - name: Install Knope
+      uses: knope-dev/action@v1
+      with:
+        version: 0.6.1 # Test before updating, breaking changes likely: https://github.com/knope-dev/action#install-latest-version
+    - name: Configure Git
+      run: |
+        git config --global user.name "${{ github.triggering_actor }}"
+        git config --global user.email "${{ github.triggering_actor}}@users.noreply.github.com"
+    - name: Display Pending Prerelease
+      run: knope prerelease --dry-run
```

Comment:
Since this is equivalent to `prepare-release` save for the `--dry-run`, thoughts on making `prerelease (dry-run)` or similar a workflow option in `prepare-release`? That way we don't need to worry about keeping both in sync.

Reply:
Thanks for calling that out. They shouldn't be totally identical. I neglected to clean up the copy-pasta. There's no need for the GitHub token or username/email config on a dry run.

Given that they are different now, and this workflow is relatively tiny, I think it's okay to keep them separate. I think the knope version drifting between the two is a valid concern, but not one we're likely to run into often. If it becomes a problem, I'd rather adapt then add complexity to the workflow that actually publishes release from the start.